lib/timescaledb/model.ex

```diff
@@ -49,78 +40,26 @@ defmodule Membrane.RTC.Engine.TimescaleDB.Model do
 
   @doc """
   Takes count and interval.
-  Deletes records in database older thant `count` * `interval`.
+  Deletes records in database older than `count` * `interval`.
   `interval` might be `"year"`, `"month"`, `"week"`, `"day"`, `"hour"`, `"minute"`, `"second"`, `"millisecond"` or `"microsecond"`.
   """
-  @spec remove_outdated_records(non_neg_integer(), String.t()) :: :ok
-  def remove_outdated_records(count, interval) do
-    for model <- [PeerMetrics, TrackMetrics, PeerToRoom, TrackToPeer] do
-      from(p in model, where: p.time < ago(^count, ^interval))
-      |> Repo.delete_all()
+  @spec remove_outdated_records(module(), non_neg_integer(), String.t()) :: :ok
+  def remove_outdated_records(repo, count, interval) do
+    for model <- [PeerMetrics, TrackMetrics] do
+      query = from(p in model, where: p.inserted_at < ago(^count, ^interval))
+      apply(repo, :delete_all, [query])
```

Comment:
There's no need to use `apply` here
```suggestion
      repo.delete_all(query)
```